src/stores/GraphStore.ts

```diff
@@ -56,6 +57,9 @@ export class GraphStore {
 	@observable
 	public interval: IntervalOption = 15;
 
+	@observable
```

Comment:
You don't need `@observable` here. Just make readonly property like this.steps and place above